project-keeper/src/main/resources/templates/.github/workflows/ci-build-db-version-matrix.yml

```diff
@@ -0,0 +1,70 @@
+# Generated by Project Keeper
+# https://github.com/exasol/project-keeper/blob/main/project-keeper/src/main/resources/templates/.github/workflows/ci-build-db-version-matrix.yml
+name: CI Build
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: $ciBuildRunnerOS
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.exasol_db_version }}
+      cancel-in-progress: true
+    strategy:
+      fail-fast: false
+      matrix:
+        exasol_db_version: [$matrixExasolDbVersions]
+    env:
+      DEFAULT_EXASOL_DB_VERSION: $defaultExasolDbVersion
+    steps:
+      - name: Free Disk Space
+        if: ${{ $freeDiskSpace }}
+        run: |
+          sudo rm -rf /usr/local/lib/android
+          sudo rm -rf /usr/share/dotnet
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 11 & 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: "temurin"
+          java-version: |
+            17
+            11
+          cache: "maven"
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v3
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+      - name: Enable testcontainer reuse
+        run: echo 'testcontainers.reuse.enable=true' > "$HOME/.testcontainers.properties"
+      - name: Run tests and build with Maven
+        run: |
+          JAVA_HOME=$JAVA_HOME_11_X64 mvn --batch-mode clean verify \
```

Comment:
Should we add an additional environment variable for Scala?
See https://github.com/exasol/spark-connector/blob/main/.github/workflows/ci-build.yml#L51

Reply:
Good idea!